src/os/bluestore/BlueStore.cc

```diff
@@ -5651,6 +5653,29 @@ ObjectMap::ObjectMapIterator BlueStore::get_omap_iterator(
 
 // -----------------
 // write helpers
+void BlueStore::_save_min_min_alloc_size(uint64_t new_val)
+{
+  assert(new_val > 0);
+  if (min_min_alloc_size > 0 &&
+      new_val >= min_min_alloc_size) {
```

Comment:
@liewegas , you meant we should put this message in both cases when
min_min_alloc_size > min_alloc_size and
min_min_alloc_size < min_alloc_size ?

Because once min_alloc_size is increased, we will keep seeing that message every time we boot, is that ok?